configure.ac

```diff
@@ -1,5 +1,5 @@
 AC_PREREQ(2.60)
-AC_INIT(ofono, 1.26)
+AC_INIT(ofono, XXX)
```

Comment:
better to keep this as it is, or maybe "1.26+", to know from which ofono version it is branched.
It looks like it is worth keeping trace also of the ell library with which the version has been tested, maybe a composite version number is more adequate.